vectordraw/templates/html/vectordraw_edit.html

```diff
@@ -0,0 +1,372 @@
+                <option value="1" {% if field.value %}selected{% endif %}>
+                  True {% if field.default %}&nbsp;&nbsp;&nbsp;&nbsp;(Default){% endif %}
+                </option>
+                <option value="0" {% if not field.value %}selected{% endif %}>
+                  False {% if not field.default %}&nbsp;&nbsp;&nbsp;&nbsp;(Default){% endif %}
+                </option>
+              </select>
+            {% elif field.type == "string" %}
+              <input type="text"
+                     class="field-data-control"
+                     id="xb-field-edit-{{field.name}}"
+                     value="{{field.value|default_if_none:""}}"
+                     aria-describedby="{{field.name}}-help">
+            {% elif field.type == "integer" or field.type == "float" %}
+              <input type="number"
+                     class="field-data-control"
+                     id="xb-field-edit-{{field.name}}"
+                     {% if field.step %} step="{{field.step}}" {% elif field.type == "integer" %} step=1 {% endif %}
+                     {% if field.max %} max="{{field.max}}" {% endif %}
+                     {% if field.min %} min="{{field.min}}" {% endif %}
+                     value="{{field.value|default_if_none:""}}"
+                     aria-describedby="{{field.name}}-help">
+            {% elif field.type == "text" or field.type == "html" %}
+              <textarea class="field-data-control"
+                        data-field-name="{{field.name}}"
+                        id="xb-field-edit-{{field.name}}"
+                        aria-describedby="{{field.name}}-help"
+                        rows=10 cols=70>{{field.value}}</textarea>
+            {% else %}
+              {% trans "Unsupported field type. This setting cannot be edited." %}
+            {% endif %}
+
+            {% if field.allow_reset %}
+              <button class="action setting-clear {% if field.is_set %}active{%else%}inactive{% endif %}"
+                      type="button"
+                      name="setting-clear"
+                      value="{% trans "Clear" %}"
+                      data-tooltip="{% trans "Clear" %}">
+                <i class="icon fa fa-undo"></i><span class="sr">{% trans "Clear Value" %}</span>
+              </button>
+            {% endif %}
+          </div>
+          {% if field.help %}
+            <span id="{{field.name}}-help" class="tip setting-help"> {{ field.help|safe }} </span>
+          {% endif %}
+        </li>
+      {% endfor %}
+      <li>
+        <!-- WYSIWYG editor -->
+        <div class="vectordraw_block vectordraw_edit_block">
+          <h2 aria-describedby="wysiwyg-description">
+            WYSIWYG Editor
+            <button class="info-button" title="Info">
+              <span class="info-label fa fa-question" aria-hidden="true"></span>
+              <span class="sr">{% trans "Toggle information about how to use WYSIWYG editor" %}</span>
```

Comment:
I think you're missing the word "the" -- "how to use *the* WYSIWYG editor". You also may want to add a translators comment explaining what WYSIWYG means.